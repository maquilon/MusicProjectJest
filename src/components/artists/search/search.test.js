import React from 'react';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<button
            className="btn btn-primary custom-btn ml-3 my-sm-0"
            type="button"
            onClick={() => this.handleOnClick()}
        > Search</button>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
