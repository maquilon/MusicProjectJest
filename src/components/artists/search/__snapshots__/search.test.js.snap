// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="btn btn-primary custom-btn ml-3 my-sm-0"
  onClick={[Function]}
  type="button"
>
   Search
</button>
`;
